common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -29,6 +29,12 @@
  * according to a specific format.
  */
 public class FormatUtils {
+  /**
+   * Parse a list of Objects into a String.
```

Comment:
Might be better to mention the format used like `the string representation returned by Object#toString, separated by comma`?